RecoLocalTracker/SiStripClusterizer/plugins/ClustersFromRawProducer.cc

```diff
@@ -444,7 +443,9 @@ void ClusterFiller::fill(StripClusterizerAlgorithm::output_t::FastFiller & recor
   if(!record.empty()) incNoZ();
 
   // COUT << "filled " << record.size() << std::endl;
-  
+} catch (edmNew::CapacityExaustedException) {
```

Comment:
Can throwing an exception in this block leave this object or objects it interacts with in a bad state?